mex/extractors/datenkompass/transform.py

```diff
@@ -75,7 +75,9 @@ def get_unit_shortname(
     Returns:
         List of short names of contact units as strings.
     """
-    return delim.join(
+    if not responsible_unit_ids:
+        return None
+    result = delim.join(
```

Comment:
```suggestion
    if not responsible_unit_ids and  result := delim.join(
```
slightly more elegant imo